tools/toolchain/scripts/stage6/install_quip.sh

```diff
@@ -103,6 +103,8 @@ case "$with_quip" in
       export QUIP_ARCH=linux_${quip_arch}_gfortran
       # hit enter a few times to accept defaults
       echo -e "${MATH_LDFLAGS} $(resolve_string "${MATH_LIBS}") \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n" | make config > configure.log
+      # Download up-to-date config.guess
+      wget http://savannah.gnu.org/cgi-bin/viewcvs/*checkout*/config/config/config.guess -O src/fox/config/config.guess -o wget.log
```

Comment:
Maybe this should better be a patch? Otherwise, please add a checksum.

Reply:
I think it would be the best to replace the `config.guess` file from 2006 in the current QUIP archive file with the most recent version from 2021.

Reply:
I would prefer to keep the upstream archives unmodified. Fortunately, they just upgraded their fox submodule: https://github.com/libAtoms/QUIP/commit/b4336484fb65b0e73211a8f920ae4361c7c353fd. So, we can just create a new archive, ie. upgrade to the latest QUIP version.